ODIN_II/regression_test/benchmark/task/keywords/defparam/simulation_result.json

```diff
@@ -115,6 +115,53 @@
         "Estimated LUTs": 12,
         "Total Node": 12
     },
+    "defparam/defparam_depth_2/no_arch": {
+        "test_name": "defparam/defparam_depth_2/no_arch",
+        "blif": "defparam_depth_2.blif",
+        "exit": 134,
+        "errors": [
+            "[OUTPUT_BLIF] Vector files differ."
+        ],
+        "warnings": [
```

Comment:
can you check these? it seems that synthesis passes, but it doesnt produce the right vector

Reply:
I think with some of the changes we made it passes synthesis now. The issue is if you look at the benchmark the defparams are in a seperate module which isn't supported. The change isn't being taken into consideration thus what's supposed to happen isn't which leads to the vector mismatch.